src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -181,13 +233,36 @@ namespace PhotovoltaicThermalCollectors {
 
         void calculate(EnergyPlusData &state);
 
+        void calculateSimplePVT(EnergyPlusData &state);
+
+        void calculateBIPVT(EnergyPlusData &state);
+
+        void calculateBIPVTMaxHeatGain(
```

Comment:
Strings should be passed by `const &` not by value. Even better, pass a `string_view`. And better still, pass an enum if that is how you are using the string as the name suggests.

Reply:
updated